src/server/http/__tests__/xsrf.js

```diff
@@ -104,14 +144,58 @@ describe('xsrf request filter', function () {
         expect(resp.payload).to.be('ok');
       });
 
-      it('rejects requests without either an xsrf or version header', async function () {
+      it('accepts requests with any allowed media type', async function () {
+        const allowedContentTypes = [
```

Comment:
HAPI request has `mime` field (https://hapijs.com/api#requests) that is basically what we need here, but I'm hesitating to use it as in no-payload case (e.g. our `DELETE` calls) it defaults to `application/json` even if `Content-Type` isn't provided. 

Or you think it's acceptable and we can go with `request.mime`?